app/src/main/java/com/example/giphyservice/data/repository/CustomCallback.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.giphyservice.data.repository
+
+import com.example.giphyservice.data.model.Gif
+
+interface CustomCallback {
```

Comment:
give better naming for that callback.
Improvement: you can make it generic, https://stackoverflow.com/a/15260988/8295472